website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+    - Set ```tags``` to ```{ "Deployment" = "PREFIX" }```
+    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::YOUR ACCOUNT ID:policy/NGAPShRoleBoundary```
+    - Add ```rds_user_password = "CumulusD3faultPassw0rd"``` and change as desired.
+    - Set ```provision_user_database``` to ```true```
+    - Set ```cluster_identifier``` to ```"PREFIX-cumulus-db"```
+
+- In rds-cluster-tf/terraform.tf
+    - Set ```bucket``` to ```"PREFIX-tf-state"```
+    - Set ```key``` to ```"PREFIX/cumulus/terraform.tfstate"```
+    - Set ```dynamodb_table``` to ```"PREFIX-tf-locks"```
+
+- Go to https://github.com/asfadmin/CIRRUS-core/blob/master/Dockerfile and download the file to the same folder as your downloaded repo and orca folder.
+  :::tip
+  Make sure that no extension is added.
+  :::
+- Open a commandline in the same folder.
+    - Run ```docker build -t orca .``` and ```docker run -it --rm -v pathToYourFolder:/CIRRUS-core orca /bin/bash```
+    - The commandline should now be inside a docker container.
+      ```bash
+      cd cumulus-orca-template-deploy/rds-cluster-tf/
+      aws configure
+      terraform init
+      terraform plan
+      terraform apply
+      ```
+
+- In data-persistence-tf/terraform.tfvars
+    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::12345:policy/NGAPShRoleBoundary``` replacing the ```12345``` with your Account Id.
+      <img alt="Location of Account Id" src={useBaseUrl('img/aws-account-id.PNG')} />
+    - Set ```rds_user_access_secret_arn``` to the ```user_credentials_secret_urn``` output from ```terraform apply```.
+    - Set ```rds_security_group``` to the ```security_group_id``` output from ```terraform apply```.
+    - Set ```vpc_id``` to your borrowed VPC.
+
+```bash
+cd ../data-persistence-tf/
+aws configure
+terraform init
+terraform plan
+terraform apply
+```
+
+- In cumulus-tf/terraform.tfvars
+    - Add the line ```aws_profile = "default"```
+    - Replace 12345 in permissions_boundary_arn with the Account Id.
+    - Add to the buckets:
+      ```
+      default_orca = {
+         name = "prefix-orca"
+         type = "orca"
+        },
+        provider = {
+          name = "orca-sandbox-s3-provider"
+          type = "provider"
+      }
+      ```
+      :::tip
+      The "orca-sandbox-s3-provider" bucket contains test data.
+      If creating a separate environment, you can create your own bucket.
+      :::
+
+    - Replace the ```cumulus_message_adapter_lambda_layer_version_arn``` with a valid one.
+
+      :::warning
+      Process for finding the correct arn is not documented.
+      :::
+-
+    - Set the ```ecs_cluster_instance_image_id``` to ```""```
+    - ```ecs_cluster_instance_subnet_ids``` and ```lambda_subnet_ids``` should have the same two values.
+    - Set ```urs_client_id``` and ```urs_client_password``` to the values from your created application.
+    - Add an extra property ```urs_url = "https://uat.urs.earthdata.nasa.gov"```
+    - Add your username to the ```api_users```
```

Comment:
May want to add all the team user names to this value so that others can utilize the instance as well if needed for testing/verification.

Reply:
Added suggestion.